### PR TITLE
Major rewrite of install docs and 7.2 updates

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [x] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Sample scripts
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Language Spec
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Preview content
- [ ] Version 7.1 content
- [ ] Version 7.0 content
- [ ] Version 5.1 content

## PR Checklist

- [ ] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [ ] PR has a meaningful title
- [ ] PR is targeted at the _staging_ branch
- [ ] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [ ] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/install/Installing-PowerShell-on-Linux.md [hunk 1]
```diff
@@ -1,943 +1,96 @@
...
-For Linux distributions that aren't officially supported, you can try to install PowerShell using
-the [PowerShell Snap Package][snap]. You can also try deploying PowerShell binaries directly using
-the Linux [`tar.gz` archive][tar], but you would need to set up the necessary dependencies based on
-the OS in separate steps.
+# Install PowerShell on Linux
 
-[snap]: #snap-package
-[tar]: #binary-archives
+PowerShell can be installed on different Linux distributions. Most Linux platforms and distributions
+have a major release each year, and most provide a package manager that is used to install
```
Comment (chasewilson) [inline reference/docs-conceptual/install/Installing-PowerShell-on-Linux.md:9]:
```suggestion
have a major release each year, and provide a package manager that is used to install
```

File: reference/docs-conceptual/install/Installing-PowerShell-on-Linux.md [hunk 2]
```diff
@@ -1,943 +1,96 @@
...
-```
+For more information, see [Install PowerShell on Ubuntu](install-ubuntu.md).
 
-### Uninstalling binary archives
+## Raspberry Pi OS
 
-```sh
-sudo rm -rf /usr/bin/pwsh /opt/microsoft/powershell
-```
+[Raspberry Pi OS][raspbian] (formerly Raspbian) is a free operating system based on Debian.
```
Comment (chasewilson) [inline reference/docs-conceptual/install/Installing-PowerShell-on-Linux.md:66]: Shouldn't there be an include for rasbian like the others?

**Review by chasewilson — APPROVED** [review]

File: reference/docs-conceptual/install/Installing-PowerShell-on-Linux.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/install/Installing-PowerShell-on-Linux.md:66]: No. They only support the current release. No versions to track.

File: reference/docs-conceptual/install/Installing-PowerShell-on-macOS.md [hunk 3]
```diff
@@ -26,8 +32,6 @@ There are several ways to install PowerShell on macOS. Choose one of the followi
 After installing PowerShell, you should install [OpenSSL](#installing-dependencies). OpenSSL is
 needed for PowerShell remoting and CIM operations.
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/install/Installing-PowerShell-on-macOS.md:33]:
```suggestion
needed for PowerShell WSMan remoting.
```

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-72.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+---
+
+# What's New in PowerShell 7.2
+
+PowerShell 7.2 is the next Long Term Servicing (LTS) release is built on .NET 6.0.
+
+PowerShell 7.2 includes the following features, updates, and breaking changes.
+
+- New universal installer packages for most supported Linux distributions
+- Microsoft Update support
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-72.md:14]:
```suggestion
- Microsoft Update support on Windows
```

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-72.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+- 3 new experimental features
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-72.md:15]: This says 3 but lists two?  The ANSI color support is non-experimental, but the ANSI FileInfo color support is experimental.  Line 16 should probably be "Improve native command argument passing support"

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-72.md (hunk 5 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-72.md:15]: Fixed

**Pull request merged**